app/models/classification.rb

```diff
@@ -206,9 +213,9 @@ def self.categories(region_id = my_region_number, ns = DEFAULT_NAMESPACE)
 
   def self.category_names_for_perf_by_tag(region_id = my_region_number, ns = DEFAULT_NAMESPACE)
     in_region(region_id).is_category.where(:perf_by_tag => true)
-      .includes(:tag)
-      .collect { |c| c.name if c.tag2ns(c.tag.name) == ns }
-      .compact
+                        .includes(:tag)
+                        .collect { |c| c.name if c.tag2ns(c.tag.name) == ns }
+                        .compact
```

Comment:
I actually prefer the original way...I don't like the mega-indent style.  I would argue that the .is_category and .where should be moved to their own lines.

Reply:
I'm find with that, but then the rubocop rule needs updating.

Reply:
Or we can just ignore rubocop :wink: